home/map.asm

```diff
@@ -1957,7 +1957,7 @@ GetMovementPermissions::
 
 .ok_down
 	ld hl, wTilePermissions
-	set 3, [hl]
+	set RIGHT, [hl]
```

Comment:
I don't understand why it's the `RIGHT` bit in all four directions, but it really seems that way. Inconsequential copy+paste bug?

Reply:
Yeah; this looks like a copy+paste bug. It appears to have been fixed/changed in pokecrystal.